doc.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2023 Google LLC
```

Comment:
Let's use the `//` throughout here, including in blank lines. It's easier to read quickly.